main.go

```diff
@@ -140,7 +154,8 @@ If the passed version does not exist, an error is signaled.`)
 - "autoalias": Apply auto aliasing to the provider.`)
 
 	cmd.PersistentFlags().StringVar(&context.UpstreamProviderName, "upstream-provider-name", "",
-		"The name of the upstream provider.")
+		`The name of the upstream provider.
+Must be set in '.upgrade-config.yml' or passed with flag.`)
```

Comment:
`MarkFlagRequired` already adds documentation saying it is required. "You can set a flag, `.upgrade-config.yaml` or pass via a flag" is true for all options. It makes sense to mention that part in the general help description, but not on an individual option.
```suggestion
		`The name of the upstream provider.`)
```